core/controllers/contributor_dashboard.py

```diff
@@ -126,7 +127,15 @@ class ContributionOpportunitiesHandler(
 
     @acl_decorators.open_access
     def get(self, opportunity_type: str) -> None:
-        """Handles GET requests."""
+        """Handles GET requests and retrieves data based on the opportunity type
+
+        Args:
+            opportunity_type: str. The opportunity type.
+
+        Raises:
+            PageNotFoundException. The opportunity type is invalid.
+            InvalidInputException. If language code is None.
```

Comment:
InvalidInputException. The language code is None. (Don't use "if" in Raises docstrings.)

(But  I don't understand this, since the language code is not passed into the method. This might need to be clarified.)

Reply:
I figured I add the exception because on line 150 the function raises an InvalidInputException if the language is None. Maybe I shouldn't put the exception since it wasn't the result of invalid arguments?